<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>335</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>335</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>335</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CubeRotation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>281</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutS" stretch="0,0,0">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QSlider" name="xSlider">
       <property name="toolTip">
        <string>X</string>
       </property>
       <property name="toolTipDuration">
        <number>1</number>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>-360</number>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
       <property name="tickInterval">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="ylSlider">
       <property name="minimum">
        <number>-360</number>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="zSlider">
       <property name="minimum">
        <number>-360</number>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="CubeWidget" name="cube">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>420</width>
      <height>250</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>444</x>
      <y>20</y>
      <width>16</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutLabels">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>14</number>
     </property>
     <item>
      <widget class="QLabel" name="xLabel">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="yLabel">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="zLabel">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>16</width>
      <height>251</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="xyzLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>91</width>
      <height>100</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/resources/XYZ.png</pixmap>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>12</y>
      <width>31</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutAngleValues">
     <property name="topMargin">
      <number>16</number>
     </property>
     <property name="bottomMargin">
      <number>30</number>
     </property>
     <item>
      <widget class="QLabel" name="xLabelValue">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="yLabelValue">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="zLabelValue">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CubeWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>cubewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
